pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -1050,6 +1051,26 @@ impl RemoteTimelineClient {
         Ok(())
     }
 
+    async fn flush_deletion_queue(&self) -> Result<(), DeletionQueueError> {
+        match tokio::time::timeout(
+            Duration::from_secs(10),
```

Comment:
nit: a named constant for this would be nice

Reply:
It's a fair cop.  Added.